my_dual_robot_cell/my_dual_robot_cell_control/config/update_rate.yaml

```
@@ -0,0 +1,3 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 125  # Hz
```

Could we please default to an e-series robot?
```suggestion
    update_rate: 500  # Hz
```